rust/lang/security/audit/Insufficient-protected-credentials.yml

```diff
@@ -0,0 +1,20 @@
+rules:
+  - id: insufficient-protected-credentials
+    message: |
+      This is a Insufficiently Protected Credentials weakness: https://cwe.mitre.org/data/definitions/522.html
+      Consider using an appropriate security mechanism to protect the credentials (e.g. keeping secrets in environment variables)
```

Comment:
```suggestion
    message: >-
      This is an Insufficiently Protected Credentials weakness: https://cwe.mitre.org/data/definitions/522.html
      Consider using an appropriate security mechanism to protect the credentials (e.g. keeping secrets in environment variables).
```

Reply:
Please use the `>-` operator for messages. This together with some other guidelines regarding metadata are available in the docs.